src/App.tsx

```diff
@@ -1,61 +1,40 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { getUserById } from './services/user';
+import { Todo } from './types/Todo';
+import { PostForm } from './PostForm';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
```

Comment:
If `getUserById` is an asynchronous function, this code will not work as expected because it assumes `getUserById` returns a user object synchronously. Consider using `async/await` or handling the promise if `getUserById` is asynchronous.